user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_update_timing_booking_silently.md

```diff
@@ -49,3 +49,8 @@ public class Script
  }
 }
 ```
+
+> [!NOTE]
+> Since SRM Framework 1.2.37/2.0.2 <!-- RN40481 --> it's possible to finish a permanent booking while defining post-roll. To do so, the timing of that booking should be changed, by changing the end date to a time in the near future and setting the appropriate post-roll time.
```

Comment:
> From version 1.2.37/2.0.2 of the SRM framework onwards<!-- RN40481 -->, if a permanent booking is changed so that it finishes at a specified time in the near future, a post-roll time can also be specified.